### PR TITLE
Merge branch 'main' into percentage

### PR DESCRIPTION
**Pull request by splendidcomputer**

### CODE REVIEW

**Comment by splendidcomputer** [comment]
Fix: adds showing < 5% for small percentages.

File: apps/web/app/cluster.tsx [hunk 1]
```diff
@@ -32,7 +32,8 @@ export function ClusterComponent({
         >
           <div className="card-body pb-0 pl-1 pr-1">
             <div className="card-title whitespace-nowrap">
-              {cluster.name} {cluster.percentage.toFixed(1)} %
+              {cluster.name}{" "}
+              {cluster.percentage >= 5 ? cluster.percentage.toFixed(1) : "<5"} %
```
Comment (moncapitaine) [inline apps/web/app/cluster.tsx:36]: use template strings ?

**Review by moncapitaine — CHANGES_REQUESTED** [review]
can you create the string more readable?

**Comment by moncapitaine** [comment]
and please resolve conflicts

**Pull request closed without merging**